FSW/src/headers/timingClass.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef TIMINGCLASS_H
+#define TIMINGCLASS_H 
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+
+/* - - - - - - Class Definitions - - - - - - */
+class ScienceMode
+{
+    protected:
+        int mode; // according to mode enum

```

Comment:
Why protect `mode`?

Reply:
I added more stuff to this class to justify it being a class instead of just an enum, in this commit you looked at it wasn't really justified tbh

Reply:
I'm just wondering why `mode` is a protected class method and not public